chainstate/tx-verifier/src/transaction_verifier/signature_check.rs

```diff
@@ -40,24 +40,25 @@ where
 
     let inputs_utxos = inputs
         .iter()
-        .map(|input| {
-            let outpoint = input.outpoint();
-            utxo_view
+        .map(|input| match input {
+            TxInput::Utxo(outpoint) => utxo_view
                 .utxo(outpoint)
                 .map_err(|_| utxo::Error::ViewRead)?
                 .ok_or(ConnectTransactionError::MissingOutputOrSpent)
-                .map(|utxo| utxo.take_output())
+                .map(|utxo| Some(utxo.take_output())),
+            TxInput::Account(_) => Ok(None),
```

Comment:
No signature verification for accounts?

Reply:
This code just collects utxos for verification, all inputs are checked below